src/main/java/com/kylenicholls/stash/parameterizedbuilds/eventHandlers/PushHandler.java

```diff
@@ -66,4 +68,35 @@ public void onEnd(ChangeSummary summary) throws IOException {
             return false;
         }
     }
+
+    boolean validateCommitMsg(Job job) {
+        String ignoreCommitMsg = job.getIgnoreCommitMsg();
+        if(ignoreCommitMsg.isEmpty()){
+            return true;
+        }
+        else{
+            CommitRequest commitRequest = new CommitRequest.Builder(
+                    repository, refChange.getToHash()).build();
+            String actualCommitMsg = commitService.getCommit(commitRequest).getMessage();
+            boolean hasIgnoreMsg = actualCommitMsg != null &&
+                    actualCommitMsg.toLowerCase().contains(ignoreCommitMsg.toLowerCase());
```

Comment:
I think it would be better to treat the ignoreCommitMsg as a regex instead of just a substring. This would give users more power to check for multiple different strings within a commit message or only check a certain portion of the string (beginning or end typically).